example/Utils.purs

```diff
@@ -0,0 +1,92 @@
+module Utils where
+
+import Control.Monad.Eff
+import Control.Monad.Eff.Random
+import DOM
+import Math (floor)
+import Data.Array
+import Data.Maybe
+import Data.Tuple
+
+foreign import undefine  """
```

Comment:
This is unsafe. Of course, I realize it's only used in the examples, but how can we do this in a safe way?